scripts/new_project.kts

```diff
@@ -241,14 +299,14 @@ object NewProject {
         showMessage("=> 🔎 Renaming application class...")
         File(projectPath)
             .walk()
-            .filter { it.name == "$TEMPLATE_APPLICATION_CLASS_NAME.kt" || it.name == "AndroidManifest.xml" }
+            .filter { it.name == "${templateApplicationClassName}.kt" || it.name == "AndroidManifest.xml" }
```

Comment:
@Tuubz @chornerman This is the old logic, however, I don't think we should rename the application class to fit the generated class name. There is no best practice to rename it. Should we name the Application class as `MainApplication` from the template & samples? 💭 

IMO, removing `Renaming application class...` logic could simplify our script logic and allow us to focus on the rest of things more efficiently.

Reply:
Sounds good, but let's keep it for another issue then. 

Preferably don't want to start including changes from `sample` in here 🙏

Reply:
@Tuubz I created an RFC [here](https://github.com/nimblehq/android-templates/discussions/357), let's close this comment.